### DISCUSSION TITLE
How do I migrate data from one Aidbox to another?

### DISCUSSION THREAD

**Discussion by Nesmeshnoy** [Aidbox/Issues#490] [Q&A]
We are currently using `prod.XXX.aidbox.app`, and we want to migrate all the data to the `staging.XXX.aidbox.app`. Is there any solution for it?

**Comment by ier** [comment]
The simplest way is the following:
1. Remove staging database
2. Backup prod database
3. Recreate staging database from prod backup

https://docs.aidbox.app/api-1/bulk-api-1/usddump

This should do the thing.
